addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1581,7 +1581,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#348"
-msgid "Enable passthrough"
+msgid "Allow passthrough"
```

Comment:
Personally I think those could lead to just as much confusion as fritsch is trying to fix